test/test_vmap.py

```diff
@@ -1150,5 +1159,100 @@ def test_no_random_op_support(self):
                                         'vmap: We do not yet support calling random operations'):
                 vmap(op)(*args)
 
+def construct_v(output, batch_size):
+    return torch.randn(batch_size, *output.shape,
+                       dtype=output.dtype, device=output.device)
+
+def as_tuple(x):
+    if isinstance(x, tuple):
+        return x
+    elif isinstance(x, list):
+        return tuple(x)
+    else:
+        return x,
```

Comment:
No action really necessary here, but we have some other code like this in other places. It reminds me a bit of https://jax.readthedocs.io/en/latest/pytrees.html too